test/dlc_tests/ecs/tensorflow/inference/test_ecs_tensorflow_inference.py

```diff
@@ -206,7 +206,7 @@ def test_ecs_tensorflow_inference_neuronx_inf2(
 
 @pytest.mark.model("half_plus_two")
 @pytest.mark.team("frameworks")
-@pytest.mark.parametrize("ecs_instance_type", ["p3.8xlarge"], indirect=True)
+@pytest.mark.parametrize("ecs_instance_type", ["g4dn.8xlarge"], indirect=True)
```

Comment:
Switched to g4dn instance type to deal with ECS instance termination timeouts when using p3.8xlarge instances.